resources/metadata/integrationArtifactDeploy.yaml

```diff
@@ -16,6 +16,7 @@ spec:
         description: User to authenticate to the SAP Cloud Platform Integration Service
         scope:
           - PARAMETERS
+          - GENERAL
```

Comment:
@CCFenner Does giving the `username` and `password` parameters the `GENERAL` scope allow for the Jenkins specific `cpiCredentialsId` parameter to be configurable in the general section of the config.yaml?

Reply:
@SAP/jenkins-library-team Does anyone know the answer to this?

Reply:
Answer: no

Credentials are fetched on any level.

Reply:
Then @mayurmohan please remove the GENERAL scope from the username and password parameters, we don't want them available in the general section of the config.yaml file.